pages/api/create-member.tsx

```diff
@@ -191,54 +186,58 @@ const addToFirebase = async (
   });
 };
 
-const sendSgEmail = async ({
-  email,
-  firebaseId,
-  name,
-}: SendConfirmationEmailProps) => {
-  return new Promise((resolve, reject) => {
-    sendConfirmationEmails({
-      email: email,
-      firebaseId: firebaseId,
-      name: name,
-    })
-      .then((response) => {
-        resolve(response);
-      })
-      .catch((error) => {
-        reject(error);
-      });
-  });
-};
-
 export default async function handler(req, res) {
   if (req.method !== "POST") {
     return res.status(405).json({ message: "Only POST requests allowed" });
   }
   try {
-    const isEmailUsed = await emailExists(req.body.email);
-    if (!isEmailUsed) {
-      const docRef: DocumentReference = await addToFirebase({
-        ...req.body,
-      }).then((body) => {
-        console.log("✅ added member to firebase");
-        return body;
-      });
-      await sendSgEmail({
-        email: req.body.email,
-        firebaseId: docRef.id,
-        name: req.body.name,
-      }).then(() => {
-        console.log("✅ sent member email via sendgrid");
-      });
-      return res.status(200).json({ message: "Successfully added member." });
-    } else {
-      return res.status(422).json({
-        error: "This email is associated with another member.",
-        body: "We only allow one member per email address.",
+    const {
+      email,
+      name,
+      location,
+      title,
+      website, // TODO: Remove "website" input param and replace with "link"
+    } = req.body;
+    console.log(req.body);
```

Comment:
nit: do we want to be logging this? I believe it would only be shown on the server side so no risk, but may add logs we don't need

Reply:
Oops, yep this was unintentional. Removing!